app/request/forms.py

```diff
@@ -268,6 +269,23 @@ def __init__(self):
             # set secondary agencies to be below the primary
             for agency in user_agencies:
                 self.agency_ein.choices.insert(2, self.agency_ein.choices.pop(self.agency_ein.choices.index(agency)))
+
+            # get choices for agency user select field
+            if current_user.is_agency_admin():
```

Comment:
This looks like duplicated code from the agency api. Can we make this into a function in `api/utils.py` and then import it in `api/views.py` and `app/request/forms.py`?

Reply:
Resolved. Created function in `api/utils.py`.